rclrs/build.rs

```diff
@@ -37,10 +37,14 @@ fn main() {
         .allowlist_var("rcutils_.*")
         .allowlist_var("rosidl_.*")
         .layout_tests(false)
-        .size_t_is_usize(true)
```

Comment:
This is true by default as well ([source](https://docs.rs/bindgen/latest/src/bindgen/options/mod.rs.html#1588))